### PR TITLE
Spam Bot Protection - Encode E-Mail Adress and make use of Smarty mailto

### PR DESCRIPTION
**Pull request by waschier-design**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       |  develop
| Description?  | Encode E-Mail Adress and make use of Smarty mailto & encode function for better spam bot protection
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9180)

### CODE REVIEW

File: themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl [hunk 1]
```diff
@@ -54,16 +54,8 @@
...
-        {l
-          s='Email us: [1]%email%[/1]'
-          sprintf=[
-            '[1]' => '<a href="mailto:'|cat:$contact_infos.email|cat:'" class="dropdown">',
-            '[/1]' => '</a>',
-            '%email%' => $contact_infos.email
-          ]
-          d='Shop.Theme.Global'
-        }
+        {l s='Email us: ' d='Shop.Theme.Global'}
```
Comment (PierreRambaud) [inline themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl:57]: Space is not needed in translation =)
```suggestion
        {l s='Email us:' d='Shop.Theme.Global'}
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by LouiseBonnard** [comment]
As @PierreRambaud said, space must be removed in the _Email us:_ translation key, otherwise it looks all fine for me. :-)

**Comment by matks** [comment]
Hi @waschier-design I'm sorry I see this PR got stale and you deleted your repository. I recreated the PR here https://github.com/PrestaShop/PrestaShop/pull/17562 and I mentioned you to give you credits. Feel free to tell me if you wish to rebuild/update this PR instead.

**Comment by matks** [comment]
@waschier-design I'm happy to tell you that, 1 year and half later, we finally merged this contribution through https://github.com/PrestaShop/PrestaShop/pull/17562

Sorry for delay, I hope this message reaches you

**Comment by waschier-design** [comment]
Wohoo... Works

**Pull request closed without merging**